docs/core/index.yml

```diff
@@ -0,0 +1,122 @@
+### YamlMime:Landing
+
+title: .NET Core documentation
+summary: Learn about .NET Core, an open-source, general-purpose development platform for building cross-platform apps.
+
+metadata:
+  title: .NET Core documentation
+  description: .NET Core is an open-source, general-purpose development platform for building cross-platform apps.
+  ms.topic: landing-page
+  ms.date: 03/24/2020
+
+# linkListType: architecture | concept | deploy | download | get-started | how-to-guide | learn | overview | quickstart | reference | sample | tutorial | video | whats-new
+
+landingContent:
+
+  # Card
+  - title: Learn about .NET Core
+    linkLists:
+      - linkListType: overview
+        links:
+          - text: About .NET Core
+            url: about.md
+          - text: NET Core SDK overview
+            url: sdk.md
+      - linkListType: whats-new
+        links:
+          - text: What's new in .NET Core 3.1
+            url: whats-new/dotnet-core-3-1.md
+          - text: What's new in .NET Core 3.0
+            url: whats-new/dotnet-core-3-0.md
+
+  # Card
+  - title: Get started with .NET Core
+    linkLists:
+      - linkListType: overview
+        links:
+          - text: Migrate from .NET Framework to .NET Core
+            url: porting/index.md
+      - linkListType: deploy
+        links:
+          - text: App publishing
+            url: deploying/index.md
+      - linkListType: get-started
+        links:
+          - text: Get started with .NET Core
+            url: get-started.md
+      - linkListType: tutorial
+        links:
+          - text: Get started with C# and Visual Studio Code
+            url: tutorials/with-visual-studio-code.md
+          - text: Create a Hello World app in Visual Studio 
+            url: tutorials/with-visual-studio.md
+          - text: Containerize a .NET Core app
+            url: docker/build-container.md
+      - linkListType: concept
+        links:
+          - text: Select which .NET Core version to use 
+            url: versions/selection.md
+
+  # Card
+  - title: Install .NET Core
+    linkLists:
+      - linkListType: how-to-guide
+        links:
+          - text: Install .NET Core SDK 
+            url: install/sdk.md
+          - text: Install .NET Core runtime
+            url: install/runtime.md
+          - text: Use a Linux package manager to install
```

Comment:
Wouldn't one of the two preceding links get you to this?

Reply:
Yes, but this is *the* most popular article under /core/ with 45K page views in February.